research/table_extraction_using_block_detection/code/image_to_block_feature_generator.py

```diff
@@ -59,7 +59,7 @@ def generate_blocks_dilation(self):
         '''
         kernel = np.ones((5, 10), np.uint8)
         _, thresh1 = cv2.threshold(self.img, 0, 1, cv2.THRESH_BINARY_INV)
-        return cv2.dilate(thresh1, kernel, iterations=5)
+        return cv2.dilate(thresh1, kernel, iterations=4)
```

Comment:
Declare iterations as a constant class variable, explain why you have chosen 4? Explain logic